services/openshift/scripts/openshift

```diff
@@ -10,13 +13,7 @@ export OPENSHIFT_DIR=${ORIGIN_DIR}/openshift.local.config/master
 export KUBECONFIG=${OPENSHIFT_DIR}/admin.kubeconfig
 
 # Get machine IP address
-ip=`ip addr show eth1 | awk 'NR==3 {print $2}' | cut -f1 -d /`
-
-# In case eth1 is not configured.
-if [ -z "$ip" ]; then
-    ip=`ip route get 8.8.8.8 | awk 'NR==1 {print $NF}'`
-fi
-
+ip=get_ip_address
```

Comment:
In this case value of ip will be 'get_ip_address' not actual IP, it should be changed to
```
ip=$(get_ip_address)
```